pkg/apps/apiserver/registry/deploylog/rest.go

```diff
@@ -59,11 +59,11 @@ var _ = rest.GetterWithOptions(&REST{})
 // one for deployments (replication controllers) and one for pods to get the necessary
 // attributes to assemble the URL to which the request shall be redirected in order to
 // get the deployment logs.
-func NewREST(dcClient appsclient.DeploymentConfigsGetter, rcClient kcoreclient.ReplicationControllersGetter, podClient corev1client.PodsGetter) *REST {
+func NewREST(dcClient appsclient.DeploymentConfigsGetter, client kubernetes.Interface) *REST {
 	r := &REST{
 		dcClient:  dcClient,
-		rcClient:  rcClient,
-		podClient: podClient,
+		rcClient:  client.CoreV1(),
```

Comment:
I recall a discussion with @sttts (I think) where the registry bits will always be internal, has something changed? @deads2k @mfojtik ?

Reply:
@soltysh registry yes, but the client that retrieves rc can be external